### ISSUE TITLE
Encoder Latent Dimensions

### ISSUE THREAD

**Issue by naythanyeo** [rhyan10/X-MACE#5]
Hi,
I noticed that the encoder in blocks.py has been defined as follows
class PermutationInvariantEncoder(torch.nn.Module):
    def __init__(self, input_dim=1, hidden_dim=16, latent_dim=128):

the default hidden dimensions and latent dimensions seem to be reversed. 

Later on when calling the AutoencoderExcitedMace model in models.py, the encoder is initialised with
perm_encoder = PermutationInvariantEncoder(latent_dim=num_permutational_invariant)

This means the default uses hidden dimensions of 16 for the encoder. May I know if this is intentional? 

I've briefly tested swapping the defaults, using a larger hidden dims for the encoder (128) did improve the latent space alignment and reconstructed energies loss. The overall loss was also able to converge faster and overall test error was about 25% lower.
